include/libtorrent/socket_type.hpp

```diff
@@ -56,9 +56,9 @@ enum class socket_type_t : std::uint8_t {
 };
 
 // return a short human readable name for types of socket
+// TODO: move to aux
 char const* socket_type_name(socket_type_t);
```

Comment:
I decided not to export this instead (and not use it), it should be private unless really necessary